ports/mimxrt/mimxrt_flash.c

```diff
@@ -82,16 +82,27 @@ status_t flash_erase_block(uint32_t erase_addr) {
 status_t flash_write_block(uint32_t dest_addr, const uint8_t *src, uint32_t length) __attribute__((section(".ram_functions")));
 status_t flash_write_block(uint32_t dest_addr, const uint8_t *src, uint32_t length) {
     status_t status;
+    uint32_t size;
```

Comment:
I think the change to this function is not supposed to be in this PR

Reply:
Surely not. That happens when working at two branches at the same time.